client/src/components/App/Content/TabManagement/TabManagement.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Paper, Tabs, Tab, Card } from '@material-ui/core';
+
+import useStyles from './TabManagementStyles';
+
+const TabManagement: React.FC = (): JSX.Element => {
+    const classes = useStyles({});
+    const defaultTab = 0;
+
+    const [currentTab, setCurrentTab] = React.useState(defaultTab);
+
+  const handleTabChange = (event: React.ChangeEvent<{}>, selectedTab: number) => {
+    setCurrentTab(selectedTab);
+  };
+
+  const tabNames = ['פרטים אישיים', 'בידוד ופרטים קליינים', 'חשיפה אפשרית וחו"ל', 'מקומות ומגעים'];
+  
+    return (
+        <Card className={classes.card} dir='rtl'>
```

Comment:
"dir='rtl' " is already defined in the theme. You can just configure it once you're using the theme and that's it.